com/zaehlwerk.go

```diff
@@ -4,21 +4,25 @@ import (
-	Einheit           mengeneinheit.Mengeneinheit     `json:"einheit,omitempty" validate:"required"`                                   // Die Einheit der gemessenen Größe
-	Zaehlerstaende    Zaehlerstaende                  `json:"zaehlerstaende,omitempty"`                                                // A list of Zaehlerstand (Non BO4E Standard)
-	Verwendungszwecke []Verwendungszweck              `json:"verwendungszwecke,omitempty"`                                             // Verwendungungszweck der Werte Marktlokation
-	Verbrauchsart     *verbrauchsart.Verbrauchsart    `json:"verbrauchsart,omitempty"`                                                 // Stromverbrauchsart/Verbrauchsart Marktlokation
+	ZaehlwerkId       string                               `json:"zaehlwerkId,omitempty" validate:"required" example:"47110815_1"`          // ZaehlwerkId ist die Identifikation des Zählwerks (Registers) innerhalb des Zählers. Oftmals eine laufende Nummer hinter der Zählernummer.
+	Bezeichnung       string                               `json:"bezeichnung,omitempty" validate:"required" example:"Zählwerk_Wirkarbeit"` // Bezeichnung ist eine zusätzliche Bezeichnung
+	Richtung          energierichtung.Energierichtung      `json:"richtung,omitempty" validate:"required"`                                  // Richtung beschreibt die Energierichtung: Einspeisung oder Ausspeisung.
+	ObisKennzahl      string                               `json:"obisKennzahl,omitempty" validate:"required" example:"1-0:1.8.1"`          // Die ObisKennzahl für das Zählwerk, die festlegt, welche auf die gemessene Größe mit dem Stand gemeldet wird. Nur Zählwerkstände mit dieser OBIS-Kennzahl werden an diesem Zählwerk registriert.
+	Wandlerfaktor     decimal.Decimal                      `json:"wandlerfaktor" validate:"required"`                                       // Der Wandlerfaktor ist der Faktor, mit dem die Zählerstandsdifferenz multipliziert wird, um zum eigentlichen Verbrauch im Zeitraum zu kommen.
+	Einheit           mengeneinheit.Mengeneinheit          `json:"einheit,omitempty" validate:"required"`                                   // Die Einheit der gemessenen Größe
+	Zaehlerstaende    Zaehlerstaende                       `json:"zaehlerstaende,omitempty"`                                                // A list of Zaehlerstand (Non BO4E Standard)
+	Verwendungszwecke []Verwendungszweck                   `json:"verwendungszwecke,omitempty"`                                             // Verwendungungszweck der Werte Marktlokation
+	Verbrauchsart     *verbrauchsart.Verbrauchsart         `json:"verbrauchsart,omitempty"`                                                 // Stromverbrauchsart/Verbrauchsart Marktlokation
+	Unterbrechbarkeit *unterbrechbarkeit.Unterbrechbarkeit `json:"unterbrechbarkeit,omitempty"`                                             // Stromverbrauchsart/Unterbrechbarkeit Marktlokation
```

Comment:
i checked, that it is consistent with https://github.com/Hochfrequenz/BO4E-dotnet/blob/6b79477cb6799d71caf3eb2de54d9a8449ee30fc/BO4E/COM/Zaehlwerk.cs#L109